src/VideoStreaming/VideoReceiver.h

```diff
@@ -152,4 +152,3 @@ protected slots:
     bool            _showFullScreen;
     VideoSettings*  _videoSettings;
```

Comment:
According to our discussion _videoSettings should not be member of VideoReceiver - this is the only one path to make VideoReceiver a bit lighter and standalone as first step to make simple test application for CI and video development.

Reply:
resolved by  #8455